tests/test_tensorflow_sparse_data_api.py

```diff
@@ -228,7 +228,7 @@ def test_sparse_except_with_diff_number_of_x_y_rows(self):
             batch_size=BATCH_SIZE,
         )
 
-        with tiledb.SparseArray(tiledb_uri_x, mode="r") as x, tiledb.SparseArray(
+        with tiledb.open(tiledb_uri_x, mode="r") as x, tiledb.open(
```

Comment:
`mode="r"` is the default; you may omit it and save the two extra lines added by `black`